front/src/routes/scene/duplicate-scene/index.js

```diff
@@ -0,0 +1,130 @@
+import { Component } from 'preact';
+import { connect } from 'unistore/preact';
+import DuplicateScenePage from './DuplicateScenePage';
+import { route } from 'preact-router';
+import { RequestStatus } from '../../../utils/consts';
+import get from 'get-value';
+
+@connect('httpClient', {})
+class DuplicateScene extends Component {
+  goBack = async () => {
+    route(`/dashboard/scene/${this.props.scene_selector}`);
+  };
+
+  getSourceScene = () => {
+    this.props.httpClient.get(`/api/v1/scene/${this.props.scene_selector}`).then(scene => {
```

Comment:
Oups sorry I didn't see this during last review, can you use async/await ? 

We try to avoid using .then because it leads to long chain of .then

Reply:
I prefer the await too :)